### PR TITLE
fix(config): clean up Zooz Zen 7x configs

### PR DESCRIPTION
**Pull request by dfederm**
This cleans up and templatizes the Zooz Zen 700 series configs.

There are more Zooz devices and other Zen devices, but I figured I needed to stop somewhere. I may apply the templating to the rest of the Zen line in a future PR.

For the vast majority of these the templating only really changed strings and made them consistent (obviously), but there are a few rare cases where the existing config was slightly wrong, mostly in the default value. I double checked with the Zoon advanced settings docs to help convince myself it was indeed a bug instead of intentional, and there was no comment in the config to tell me otherwise.

In general I tried matching the style guidelines as best I could. There were a few judgement calls on whether a description was required to help elaborate.

Zooz reference docs:
* [ZEN71](https://www.support.getzooz.com/kb/article/535-zen71-on-off-switch-700-advanced-settings/)
* [ZEN72](https://www.support.getzooz.com/kb/article/538-zen72-dimmer-switch-700-advanced-settings/)
* [ZEN73](https://www.support.getzooz.com/kb/article/639-zen73-on-off-toggle-switch-700-advanced-settings/)
* [ZEN74](https://www.support.getzooz.com/kb/article/648-zen74-toggle-dimmer-700-advanced-settings/)
* [ZEN76](https://www.support.getzooz.com/kb/article/549-zen76-s2-on-off-switch-700-advanced-settings/)
* [ZEN77](https://www.support.getzooz.com/kb/article/550-zen77-s2-dimmer-700-advanced-settings/)

### CODE REVIEW

**Comment by dfederm** [comment]
@zwave-js-bot rename commit

**Comment by dfederm** [comment]
@zwave-js-bot help

**Comment by AlCalzone** [comment]
`fix` should be lowercase in the commit message - I need to improve that check label :)

**Comment by dfederm** [comment]
> `fix` should be lowercase in the commit message

I'm sure it's not a top pri, but might be nice for the bot to tell the author what they did wrong, or have some snippet in the dev docs to help craft proper PRs. Bit of a learning curve to figure out the correct process/style :)

File: packages/config/config/devices/0x027a/zen71.json [hunk 1]
```diff
@@ -76,21 +76,36 @@
 			]
 		},
 		"3": {
-			"label": "Auto Turn-Off Timer (Minutes)",
-			"description": "How long the switch remains on",
+			"label": "Auto Turn-Off Timer",
+			"description": "Set the number of minutes to automatically turn off switch once it has been turned on (0 to disable).",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen71.json]: I've asked @blhoward2 for a review, but I can tell you already that he'll ask you to remove the `(0 to disable)` part. That's what we have `"options"` for.

Comment (dfederm) [reply packages/config/config/devices/0x027a/zen71.json]: Sure no problem!

File: packages/config/config/devices/0x027a/zen71.json [hunk 2]
```diff
@@ -76,21 +76,36 @@
 			]
 		},
 		"3": {
-			"label": "Auto Turn-Off Timer (Minutes)",
-			"description": "How long the switch remains on",
+			"label": "Auto Turn-Off Timer",
+			"description": "Set the number of minutes to automatically turn off switch once it has been turned on.",
```
Comment (blhoward2) [inline packages/config/config/devices/0x027a/zen71.json]: This description is unnecessary. Please remove them throughout the files if the label makes clear what it does. We already define it is in minutes via the displayed unit. This is a broader problem in the original files, so if you can help us clean that up in these we appreciate it.

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Minor changes, otherwise looks good! We appreciate your effort.

File: packages/config/config/devices/0x027a/zen71.json (hunk 2 above)
Comment (dfederm) [reply packages/config/config/devices/0x027a/zen71.json]: Sure. I'll just go all out and templatize all these, at least the Zen 7x line, maybe the whole Zen line if I feel like putting the time in.

**Comment by dfederm** [comment]
@blhoward2 at the risk of being confusing, I renamed the PR and rewrote the description to better match the new scope of the PR. Please review at your convenience.

**Comment by dfederm** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
@dfederm FYI, force-pushes make it confusing because it doesn't allow us to see what changed from that last review.

**Comment by dfederm** [comment]
> @dfederm FYI, force-pushes make it confusing because it doesn't allow us to see what changed from that last review.

Thanks, will keep that in mind. Professionally I use a code review tool which diffs PR iterations regardless of commits so I'm used to just force-pushing to keep rebasing easy :)

**Comment by AlCalzone** [comment]
Oh cool, what tool is that?

**Comment by dfederm** [comment]
It's called CodeFlow, but it's unfortunately Microsoft-internal. I assume they haven't made it a public product in favor of improving the Azure DevOps PR experience (which also has this iteration diff feature) and hopefully that'll trickle to GitHub PRs at some point too.

**Comment by dfederm** [comment]
@blhoward2 Any other feedback or other PRs we're waiting to sequence this behind?

**Comment by blhoward2** [comment]
I've had some health issues so have been down for a couple of weeks. I'm hoping this weekend.

**Comment by dfederm** [comment]
Sorry to hear that! Hope everything is OK. Take your time

**Review by blhoward2 — DISMISSED** [review]
Looks pretty good. I made some changes for consistency. Note that this requires #3466 to be merged first as I'm reusing a some new master template entries from that PR. So lint will fail until then.

**Comment by dfederm** [comment]
@blhoward2 #3466 was merged and I resolved conflicts and removed a couple templates I introduced which became unused after your changes. Should be ready for your final stamp of approval.

**Comment by AlCalzone** [comment]
#3100 was just merged, so I've gone ahead and updated your files to match the new config format. As you'll see, paramInformation is now an array of objects.

**Review by blhoward2 — APPROVED** [review]

**Comment by dfederm** [comment]
@AlCalzone this looks ready for merge

**Pull request merged**